CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ v4.1
 ====
 - Added `OrientationsReference` as the frame orientation analog to the location of experimental markers. Enables experimentally measured orientations from wearable sensors (e.g. from IMUs) to be tracked by reference frames in the model. A correspondence between the experimental (IMU frame) orientation column label and that of the virtual frame on the `Model` is expected. The `InverseKinematicsSolver` was extended to simultaneously track the `OrientationsReference` if provided. (PR #2412)
 - Removed the undocumented `bool dumpName` argument from `Object::dump()` and made the method `const` so it can be safely called on `const` objects. (PR #2412)
+- `MarkersReference` convenience constructors were updated to take a const pointer to a `MarkerWeightSet` as its second argument. If a `Set` is provided, then only the markers listed are used as reference signals. That means `InverseKinematicsTool` no longer tracks all experimental markers even those not in the `MarkerWeightSet`. One can track all experimental markers (that have a corresponding model marker) by providing a nullptr or pointer to an empty `Set`, in which case all markers are assigned the default weight (typically 1.0).
 
```

Comment:
I'd also note somewhere that the interface for MarkersReference has changed.

Reply:
The text currently describes changes to the `MarkersReference`. What more of the interface change would you like to see. There is no reason for API users to `MarkersReference::loadMarkersFromFile` which is a convenience for `InverseKinematicsTool`.

Reply:
Let's see where the API ends up then update accordingly, I wouldn't detail passing a pointer and nullptr since this is very c++ specific. and rather low level, I would describe the high level old vs new behavior, and reference the bug that triggered this as reported by @jimmyDunne